src/Model/City.php

```diff
@@ -34,25 +34,27 @@ public function __construct(string $name, string $country, int $population, floa
 		$this->area       = $area;
 	}
 
-	public function addNewParking(): void
+	public function addNewParking(ParkingInterface $parking): void
     {
-        //TODO: implement
+        $this->parkings = $parking;
     }
 
-    public function removeParking(): void
+    public function removeParking(ParkingInterface $parking): void
     {
-        //TODO: implement
+		$key = array_search($parking, $this->parkings);
```

Comment:
Remove nie zadziała Ci po użyciu addNewParking, ponieważ $this->parkings nie będzie tablicą.